editable-templates/mvp-reference-pages/embed-html.html

```diff
@@ -0,0 +1,69 @@
+---
+title: Embed Html component
+hide_breadcrumb: false
+date_modified: 2024-10-31
+layout: no-container
+---
+
+<div class="">
+	<div class="container">
+		<div class="">
+			<div class="row">
+				<div class="title col-lg-12 col-xs-12">
```

Comment:
For maintenance: considering the custom AEM class are not prefixed anymore, we will need a registry will all those "reserved" class in order to prevent future conflict

Here I am referring to the CSS class name "title".